packaging/CMakeLists.txt

```diff
@@ -23,12 +23,18 @@ set(Halide_INSTALL_TOOLSDIR "${CMAKE_INSTALL_DATADIR}/tools"
 
 target_sources(Halide_Generator INTERFACE $<INSTALL_INTERFACE:${Halide_INSTALL_TOOLSDIR}/GenGen.cpp>)
 
-install(TARGETS Halide Halide_Generator Halide_LanguageOptions
+set(extra_targets_to_export )
+if (WITH_PYTHON_BINDINGS)
+    set(extra_targets_to_export Halide_PyStubs)
+endif ()
+
+install(TARGETS Halide Halide_Generator Halide_LanguageOptions ${extra_targets_to_export}
         EXPORT Halide_Targets
         RUNTIME COMPONENT Halide_Runtime
         LIBRARY COMPONENT Halide_Runtime
         NAMELINK_COMPONENT Halide_Development
         ARCHIVE COMPONENT Halide_Development
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
This is the default, so it's redundant.

Reply:
I think that CMake complained if LIBRARY DESTINATION was missing, or at least it did at one point, let me recheck